### PR TITLE
Added iso constants

### PR DESCRIPTION
**Pull request by linh-transcend**
## Related Issues

- https://transcend.height.app/T-16694 
- Adds iso constants so we can start encoding region information in our data map

## Security Implications

_[none]_

## System Availability

_[none]_

### CODE REVIEW

File: package.json
Comment (michaelfarrell76) [inline package.json:28]: i think you may need to add the fp-ts lib as well, its a peer dep

File: src/isoConstants/iso3166-1.ts
Comment (michaelfarrell76) [inline src/isoConstants/iso3166-1.ts:255]: should we make an enum for the country codes here? or just export a variable thats got the types of the enum?

File: package.json
Comment (linh-transcend) [reply package.json:24]: This is prob a dumb question but what part of the number to bump? It's now, `"version": "4.1.0"`. Just move it to 4.1.1?

Comment (anotherminh) [reply package.json:24]: @linh-transcend yeah that's fine!

File: src/isoConstants/iso3166-1.ts
Comment (anotherminh) [inline src/isoConstants/iso3166-1.ts:6]: linh, you can use `makeEnum` and use type overrides to get this to work as an enum as well https://github.com/transcend-io/main/blob/d7737f8881c2d22ab4c8c21be702ce1fdaaaee24/backend-support/aws-scripts/src/enums.ts#L18

File: src/isoConstants/iso3166-1.ts
Comment (anotherminh) [reply src/isoConstants/iso3166-1.ts:255]: this is exported in isoRegion below as an enum

File: src/isoConstants/iso3166-1.ts
Comment (anotherminh) [reply src/isoConstants/iso3166-1.ts:6]: nvm i see you already do this in the `isoRegion.ts` file -- just make sure to include the type overrides as well!

Comment (linh-transcend) [reply src/isoConstants/iso3166-1.ts:6]: From my understanding of the convo [here](https://github.com/transcend-io/main/pull/17237#discussion_r942758707), we're splitting region into two columns, one for country and other subdivision, so it's alright just to export the two individual enums? Is there still any need to have the intersection between the two enums? I can just remove if not needed anymore

Comment (anotherminh) [reply src/isoConstants/iso3166-1.ts:6]: hm sure - let's split them but make sure that whenever you use `makeEnum` that you also do the type override (linked above) so that we have them as enums as not just as an object

**Review by anotherminh — COMMENTED** [review]
links https://transcend.height.app/T-16694

File: src/isoRegion.ts
Comment (anotherminh) [inline src/isoRegion.ts:23]: You can remove the IsoRegion but i would keep these + include their type overrides

Comment (linh-transcend) [reply src/isoRegion.ts]: Just removed the IsoRegion and added type overrides!

File: yarn.lock
Comment (linh-transcend) [inline yarn.lock:321]: Since I removed `IsoRegion`, these two packages are no longer needed so I did a `yarn remove` for both packages. They were removed in the package.json but not here. Is this sus? @anotherminh @michaelfarrell76

**Review by anotherminh — DISMISSED** [review]
👏

File: src/isoConstants/iso3166-1.ts
Comment (michaelfarrell76) [reply src/isoConstants/iso3166-1.ts:255]: sik

File: package.json
Comment (michaelfarrell76) [reply package.json]: i normally bump the minor version (middle number) for new backwards-compat features like this

**Comment by linh-transcend** [comment]
I didn't know you could dismiss approvals! Can I get another approval? @anotherminh

**Review by anotherminh — DISMISSED** [review]
Can you bump the minor version instead of patch like mike said? (4.2.0 instead of 4.1.1)

**Review by anotherminh — APPROVED** [review]
yeah that's bc this is an open-sourced repo and we don't want people to merge new changes after the approval

**Pull request merged**
